Examples/Physics_applications/beam-beam_collision/inputs

```diff
@@ -0,0 +1,242 @@
+#################################
+########## MY CONSTANTS #########
+#################################
+my_constants.mc2 = m_e*clight*clight
+my_constants.nano = 1.0e-9
+my_constants.GeV = q_e*1.e9
+
+# BEAMS
+my_constants.beam_energy = 125.*GeV
+my_constants.beam_uz = beam_energy/(mc2)
+my_constants.beam_charge = 0.14*nano
+my_constants.sigmax = 10*nano
+my_constants.sigmay = 10*nano
+my_constants.sigmaz = 10*nano
+my_constants.beam_uth = 0.1/100.*beam_uz
+my_constants.n0 = beam_charge / (q_e * sigmax * sigmay * sigmaz * (2.*pi)**(3./2.))
+my_constants.omegab = sqrt(n0 * q_e**2 / (epsilon0*m_e))
+my_constants.mux = 0.0
+my_constants.muy = 0.0
+my_constants.muz = -0.5*Lz+3.2*sigmaz
+
+# BOX
+my_constants.Lx = 100.0*clight/omegab
+my_constants.Ly = 100.0*clight/omegab
+my_constants.Lz = 180.0*clight/omegab
+
+# for a full scale simulation use: nx, ny, nz = 512, 512, 1024
+my_constants.nx = 64
+my_constants.ny = 64
+my_constants.nz = 128
+
+
+# TIME
+my_constants.T = 0.7*Lz/clight
+my_constants.dt = sigmaz/clight/10.
+
+# DIAGS
+my_constants.every_red = 1.
+warpx.used_inputs_file = warpx_used_inputs.txt
+
+#################################
+####### GENERAL PARAMETERS ######
+#################################
+stop_time = T
+amr.n_cell = nx ny nz
+amr.max_grid_size = 128
+amr.blocking_factor = 2
+amr.max_level = 0
+geometry.dims = 3
+geometry.prob_lo = -0.5*Lx -0.5*Ly -0.5*Lz
+geometry.prob_hi =  0.5*Lx  0.5*Ly  0.5*Lz
+
+#################################
+######## BOUNDARY CONDITION #####
+#################################
+boundary.field_lo = PEC PEC PEC
+boundary.field_hi = PEC PEC PEC
+boundary.particle_lo = Absorbing Absorbing Absorbing
+boundary.particle_hi = Absorbing Absorbing Absorbing
+
+#################################
+############ NUMERICS ###########
+#################################
+warpx.do_electrostatic = relativistic
+warpx.const_dt = dt
+warpx.grid_type = collocated
+algo.particle_shape = 3
+algo.load_balance_intervals=100
+algo.particle_pusher = vay
+
+#################################
+########### PARTICLES ###########
+#################################
+particles.species_names = beam1 beam2 pho1 pho2 ele_nlbw1 pos_nlbw1 ele_nlbw2 pos_nlbw2 pho ele pos
```

Comment:
As a follow-up PR, maybe we could reduce this to only 6 species:
- Electrons, positrons and photons propagating to the right (e.g. `ele1 pos1 pho1`)
- Electrons, positrons and photons propagating to the left (e.g. `ele2 pos2 pho2`)
Having fewer species also means that we will have fewer separate Poisson solves, which could result in faster execution.

The QED processes would be:
- Quantum synchrotron between species propagating to the right: `ele1 -> pho1`, `pos1 -> pho1`
- Quantum synchrotron between species propagating to the left: `ele2 -> pho2`, `pos2 -> pho2`
- Breit-Wheeler between species propagating to the right: `pho1 -> ele1 + pos1`
- Breit-Wheeler between species propagating to the left: `pho2 -> ele2 + pos2`